internal/ihop/definition.go

```diff
@@ -92,13 +92,39 @@ type DefinitionDeprecated struct {
 
 // Arguments converts the Args map into a slice of strings of the form
 // key=value.
-func (i DefinitionImage) Arguments() []string {
+func (i DefinitionImage) Arguments() ([]string, error) {
 	var args []string
 	for key, value := range i.Args {
-		args = append(args, fmt.Sprintf("%s=%s", key, value))
+		var v string
+
+		switch valTyped := value.(type) {
+		case string:
+			v = valTyped
+		case int, int64, int32, int16, int8:
+			v = fmt.Sprintf("%d", valTyped)
+		case []string:
+			v = strings.Join(valTyped, " ")
+		case []any:
+			typedSlince := make([]string, len(valTyped))
```

Comment:
I assume that this is a typo and should read: `typedSlice`?

Reply:
yup, typo has been fixed :)